lib/livebook/apps.ex

```diff
@@ -209,4 +213,23 @@ defmodule Livebook.Apps do
       false
     end
   end
+
+  defp deploy_app(nil, _, notebook, warnings, _), do: deploy(notebook, warnings: warnings)
+
+  defp deploy_app(id, verified_hub_id, notebook, warnings, path) do
+    case Livebook.Hubs.get_offline_hub(id) do
```

Comment:
Now that we have `apps_path_hub_id`, it's not about the offline hub anymore, we just want to ensure that (if present) `apps_path_hub_id == verified_hub_id`. This automatically works with the offline hub when the user configures both `LIVEBOOK_APPS_PATH_HUB_ID` and `LIVEBOOK_TEAMS_*`.

Reply:
You right, good call!